src/index.js

```diff
@@ -50,6 +57,24 @@ const MODES = {
     // when upgrading
     manageAppLifecycle: false,
   },
+  satellite: {
+    name: 'satellite',
+    deployments: '/srv/software',
+    appsToDeploy: ['medic-api'],
+    upgradeDocuments: [`_design/medic`],
+    stageDeployment: false,
+
+    // unstaged deployments are triggered by the ddoc and then track their deployment progress in a new document
+    getWritableDeployDoc: doc => ({
```

Comment:
I'm unstoked about this interface. I'll change it to `writeLocalDeployLog: true`? Or should I just check something super generic like `mode.name === 'satellite'`?